containers/README.md

```diff
@@ -0,0 +1,52 @@
+Subscription Manager & Containers
+=================================
+
+This directory contains scripts and configuration files that are necessary for running
+subscription-manager in a containerized environment. This environment could be created
+using podman. Why do we care about running subscription-manager in a containerized
+environment? We believe that it could be useful for testing of subscription-manager.
+We also support running and testing of subscription-manager using vagrant, but running
+subscription-manager in VM could be a bit cumbersome. Running subscription-manager
+using podman could be more flexible for developers too.
+
+Requirements
+------------
+
+To create and use a containerized environment it is necessary to have following packages
+installed in the system:
+
+ * podman
+
+   $ sudo dnf install -y podman
+
+Create a containerized environment
+----------------------------------
+
+To create a containerized environment you can simply run following script:
+
+    $ podman build -f ./containers/Containerfile.ci --build-arg UID="$(id -u)" -t subman
+    $ podman run -it --rm -v /run/user/$UID/bus:/tmp/bus subman /bin/bash
+
+The above will create a clean sandbox in which to run tests reliably and
+test out subscription-manager.
+
+To do development locally and have the changes reflected locally (inside the con
+tainer) do the following run command from the base subscription-manager source
+directory:
+
+    $ podman run -it --rm -v /run/user/$UID/bus:/tmp/bus -v $PWD:/home/jenkins/subman:Z -w /home/jenkins/subman subman /bin/bash
```

Comment:
I get the following error:
Error: lsetxattr /home/wpoteat/subscription-manager/src/rhsmlib/__pycache__/client_info.cpython-39.pyc: operation not permitted